data/nav.yml

```diff
@@ -302,6 +302,12 @@
           path: "pipelines/incoming-webhooks"
         - name: "Audit log"
           path: "pipelines/audit-log"
+    - name: "Governance"
+      children:
+        - name: "Overview"
+          path: "governance/overview"
+        - name: "Pipeline templates"
+          path: "governance/pipeline-templates"
```

Comment:
@mbelton-buildkite wouldn't an equivalent of the Deployments section be something like this?:
```suggestion
    - name: "Governance"
      children:
        - name: "Overview"
          path: "governance"
        - name: "Pipeline templates"
          path: "governance/pipeline-templates"
```

Is the reason for your suggestion that you are uncertain if we will keep the "Governance" section?

I'm not too worried about what the section is used, my main concern is having a known URL we can link to from the app (we link to the page from several places).

I think I'll defer to @ozdenyilmaz for helping make a decision here, as she had proposed the URLs I've used here.

Reply:
Ah yip, what I'm suggesting is a mix between the Deployments and Security sections. 

And yes, I'd like to avoid including "governance" in the URL because it's not clear to me that it's the right section/label. I'm happy to use it in this change, but want to make it easier for us to update later if we need to.

Reply:
To help the Overview page feel more meaningful, I think it would be good to include a few more pages in the section. Are there any existing pages that would fit there?

The candidates I see are:

- [Build exports](https://buildkite.com/docs/pipelines/build-exports)
- [Build retention](https://buildkite.com/docs/pipelines/build-retention)
- [Public pipelines](https://buildkite.com/docs/pipelines/public-pipelines)

What do you think?

Reply:
As far as I understand this just changes the url to be governance/templates instead of governance/pipeline-templates. I am more than ok with that change, makes sense since we're in pipelines as a section.

Reply:
build exports definitely belong here, I am not sure about retention because there is a bigger piece of archiving builds.

not sure about public pipelines also

Reply:
> As far as I understand this just changes the url to be governance/templates instead of governance/pipeline-templates. I am more than ok with that change, makes sense since we're in pipelines as a section.

Clarified with @ozdenyilmaz that the change is to "pipelines/templates" not "governance/templates" - but she is happy with that so I'll go ahead and make that URL change @mbelton-buildkite.

(I won't make any changes around moving other things into the "Governance" section, for now at least).

Reply:
@mbelton-buildkite updated with [1c909b0](https://github.com/buildkite/docs/pull/2255/commits/1c909b0a5d60322cca9cf0a080d0951673a2341f)

Reply:
All good! I've fixed the conflict and will merge into this branch when the tests pass.

I'll propose moving the Build exports page into here in another PR.